### PR TITLE
Categorize Ship Part pedia articles

### LINKED ISSUE #1346: Categorize Ship Part pedia articles
Ship Parts need articles for pedia categories added and definitions assigned to them.

See [forum thread](http://www.freeorion.org/forum/viewtopic.php?f=6&t=10343) for full listing.

### PR DESCRIPTION
**Pull request by dbenage-cx**
Resolves #1346 

Disabled definitions were added for Industry, Research, "Trade", and Production Location parts
(no stringtable description added for "trade")

### CODE REVIEW

File: default/scripting/ship_parts/TroopPods.focs.txt [hunk 1]
```diff
@@ -6,7 +6,7 @@ Part
     mountableSlotTypes = [External Internal]
     buildcost = 3 * [[FLEET_UPKEEP_MULTIPLICATOR]]
     buildtime = 1
-    tags = "COMFORTABLE"
+    tags = [ "COMFORTABLE" "PEDIA_PC_TROOPS" ]
```
Comment (dbenage-cx) [inline default/scripting/ship_parts/TroopPods.focs.txt:9]: Is the `COMFORTABLE` tag for future addition? I find no reference to it.

Comment (geoffthemedio) [reply default/scripting/ship_parts/TroopPods.focs.txt:9]: was for test purposes, and is probably still unused

File: default/stringtables/en.txt [hunk 2]
```diff
@@ -7917,6 +7920,11 @@ invalid ship part class
 PC_DIRECT_WEAPON
 Direct Weapon
 
+PC_DIRECT_WEAPON_DESC
+'''Direct-fire weapons are weapon platforms which target without third-party assistance (a friendly ship or planet). Such weapons are prone to damage reduction from [[PC_SHIELD]].
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: this is oddly and unnecessarily verbose

File: default/stringtables/en.txt [hunk 3]
```diff
@@ -7935,57 +7943,110 @@ Launch Bay
 
 PC_FIGHTER_BAY_DESC
 '''Launch bays are the access point for [[encyclopedia FIGHTER_TECHS]] to enter or exit a ship.
-Each launch bay allows a number of fighters to launch each combat round.
-'''
+Each launch bay allows a number of fighters to launch each combat round.  Launched fighters take one combat round to reach firing range.'''
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: ambiguous. say fighters can attack the round after they are launched, and any later rounds until a combat ends

File: default/stringtables/en.txt [hunk 4]
```diff
@@ -7935,57 +7943,110 @@ Launch Bay
...
 
 # Ship part classes
 PC_SHIELD
 Shields
 
+PC_SHIELD_DESC
+'''Shield parts reduce the [[encyclopedia DAMAGE_TITLE]] sustained from each hit by the strength rating of the shield part. Only the remainder of a weapons damage value will affect the ship. Not all sources of damage will be reduced by a shield, for example [[encyclopedia FIGHTER_TECHS]] are unaffected by ship shields.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: "... from direct fire weapon hit by the shield part's strength."

lose the "Only ... "sentence.

Just say "Fighter damage is unaffected by shields."

File: default/stringtables/en.txt [hunk 5]
```diff
@@ -7935,57 +7943,110 @@ Launch Bay
...
+
+A ship shield is not reduced from normal weapons fire, each attack is reduced by the shield value. With a shield strength of 3, 2 shots from [[PC_DIRECT_WEAPON]]s will inflict 1 point of damage each (2 total).'''
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: this paragraph is confusing

File: default/stringtables/en.txt [hunk 6]
```diff
@@ -7935,57 +7943,110 @@ Launch Bay
...
+
 # Ship part classes
 PC_ARMOUR
 Armor
 
+PC_ARMOUR_DESC
+Armor parts expand the [[metertype METER_STRUCTURE]] of a ship.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: expand -> increase

Add something like "Higher structure allows a ship to sustain more damage before being destroyed."

File: default/stringtables/en.txt [hunk 7]
```diff
@@ -7935,57 +7943,110 @@ Launch Bay
...
+
 # Ship part classes
 PC_TROOPS
 Ground Troops
 
+PC_TROOPS_DESC
+'''Troop parts integrate with a ship and increase the [[metertype METER_TROOPS]] on-board, which can be used to invade enemy planets. To successfully invade a planet, the total number of invading [[metertype METER_TROOPS]] must exceed those defending (ties are awarded to the defender).
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: lose "integrate with a ship"

clarify "the total number of troops from all invading ships"

File: default/stringtables/en.txt [hunk 8]
```diff
@@ -7935,57 +7943,110 @@ Launch Bay
...
+
+An invading ship is destroyed as part of the landing operation in order to provide safe passage for troops on board. Any additional parts on the ship are lost as well.'''
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: lose "in order to provide safe passage..."

File: default/stringtables/en.txt [hunk 9]
```diff
@@ -7935,57 +7943,110 @@ Launch Bay
...
+
 # Ship part classes
 PC_DETECTION
 Detection
 
+PC_DETECTION_DESC
+Detection parts increase the [[metertype METER_DETECTION]] for the ship they are equipped on. (An objects [[metertype METER_STEALTH]] must fall within the empires [[metertype METER_DETECTION_STRENGTH]] value to be visible).
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: specify detection range

File: default/stringtables/en.txt [hunk 10]
```diff
@@ -7935,57 +7943,110 @@ Launch Bay
...
+If combat occurs in a system, ships that are invisible to an enemy are not targetted by their weapons. If the ship attacks, it may become visible for the following rounds.
+
+See also: [[metertype METER_DETECTION]], [[metertype METER_DETECTION_STRENGTH]]'''
+
 # Ship part classes
 PC_FUEL
 Fuel
 
+PC_FUEL_DESC
+'''Fuel parts affect a ships [[metertype METER_FUEL]]. Such parts might expand the maximum fuel capacity or increase the amount of fuel the ship can scavage.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: expand -> increase
scavage -> scavenge or regenerate

File: default/stringtables/en.txt (hunk 9 above)
Comment (dbenage-cx) [reply default/stringtables/en.txt]: `METER_DETECTION` resolves to `Detection Range`, unless you had some other specification in mind?

File: default/stringtables/en.txt [hunk 11]
```diff
@@ -7935,57 +7943,113 @@ Launch Bay
...
-Each launch bay allows a number of fighters to launch each combat round.
-'''
+Launched fighters join combat the round after being launched, returning once combat ends for that turn.'''
 
 # Ship part classes
 PC_SHIELD
 Shields
 
+PC_SHIELD_DESC
+'''Shield parts reduce the [[encyclopedia DAMAGE_TITLE]] sustained from each weapon hit by the shield parts strength.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt:7953]: part's

File: default/stringtables/en.txt [hunk 12]
```diff
@@ -7935,57 +7943,113 @@ Launch Bay
...
+
+Unlike a planet shield, a ship shield is a reflective shield and does not sustain any loss in strength from an attack. Ship shields are ineffective against [[encyclopedia FIGHTER_TECHS]].'''
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: "is a reflective shield" is misleading... just say something like "Ship shields are not affected by weapon hits, and reduce all subsequent direct fire weapon hits' damage by the same amount"

File: default/stringtables/en.txt [hunk 13]
```diff
@@ -7935,57 +7943,113 @@ Launch Bay
...
+
 # Ship part classes
 PC_ARMOUR
 Armor
 
+PC_ARMOUR_DESC
+Armor parts expand the [[metertype METER_STRUCTURE]] of a ship, allowing a ship to sustain more damage before being destroyed.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: expand->increase

File: default/stringtables/en.txt [hunk 14]
```diff
@@ -7935,57 +7943,113 @@ Launch Bay
...
+An invading ship is destroyed as part of the landing operation (any additional parts on the ship are lost as well).'''
+
 # Ship part classes
 PC_DETECTION
 Detection
 
+PC_DETECTION_DESC
+'''Detection parts increase the [[metertype METER_DETECTION]] for the ship they are equipped on.
+
+An objects [[metertype METER_STEALTH]] must fall within the empires [[encyclopedia DETECTION_TITLE]] value to be visible.'''
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: An object's

An empire's detection strength must equal or exceed an object's stealth to detect the object.

File: default/stringtables/en.txt [hunk 15]
```diff
@@ -7935,57 +7943,113 @@ Launch Bay
...
+
 # Ship part classes
 PC_STEALTH
 Stealth
 
+PC_STEALTH_DESC
+'''Stealth parts increase the [[metertype METER_STEALTH]] of a ship. The effects from these parts do not complement each other, only the highest rated part will affect the ship.
+
+If combat occurs in a system, ships that are invisible to an enemy are not targetted by their weapons. If the ship attacks, it may become visible for the following rounds.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: by the enemy's weapons

File: default/stringtables/en.txt [hunk 16]
```diff
@@ -7935,57 +7943,113 @@ Launch Bay
...
+
+See also: [[metertype METER_DETECTION]], [[encyclopedia DETECTION_TITLE]]'''
+
 # Ship part classes
 PC_FUEL
 Fuel
 
+PC_FUEL_DESC
+'''Fuel parts affect a ships [[metertype METER_FUEL]]. Such parts might increase the maximum fuel capacity or increase the amount of fuel the ship can scavenge or regenerate.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: ship's

File: default/stringtables/en.txt [hunk 17]
```diff
@@ -7935,57 +7943,113 @@ Launch Bay
...
+
+A ship requires a fuel value of 1.0 or more to leave a system.'''
+
 # Ship part classes
 PC_COLONY
 Colonization
 
+PC_COLONY_DESC
+'''Colony parts integrate with a ship to provide the necessary equipment to establish ownership on a new planet.
+These parts may also increase the [[metertype METER_POPULATION]] for a particular [[encyclopedia ENC_SPECIES]] on a ship, which will be used to start a colony for that species.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: this sentence is confusing... it seems to suggest that colony parts will work differently depending on the species on the ship

File: default/stringtables/en.txt [hunk 18]
```diff
@@ -7935,57 +7943,113 @@ Launch Bay
...
+
+A ship lacking full colony support will still have enough personnel to establish an [[encyclopedia OUTPOSTS_TITLE]].
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: "lacking full colony support" ? don't know what that means

File: default/stringtables/en.txt [hunk 19]
```diff
@@ -7935,57 +7943,113 @@ Launch Bay
...
+(Outposts may construct any available species colony, this combination is cheaper than a populated colony ship)
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: colony production requires supply access to the species, on planets meeting other criteria

, to .

File: default/stringtables/en.txt [hunk 20]
```diff
@@ -7935,57 +7943,113 @@ Launch Bay
...
+'''Bombardment parts allow a ship to bombard an enemy planet, typically reducing the population on the planet by the capacity value of any applicable bombardment parts on board.
+
+The majority of bombardment parts only affect one particular [[encyclopedia METABOLISM_TITLE]].'''
+
 # Ship part classes
 PC_INDUSTRY
 Industry
 
+PC_INDUSTRY_DESC
+Industry parts increase the [[metertype METER_INDUSTRY]] generated from a ship. The ship must be [[metertype METER_SUPPLY]] connected to a [[OBJ_PROD_CENTER]] (such as a colony) for the output to be utilized. It may also require [[metertype METER_POPULATION]] to begin increasing the output.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: What is "It"?

File: default/stringtables/en.txt [hunk 21]
```diff
@@ -7935,57 +7943,113 @@ Launch Bay
...
+
 # Ship part classes
 PC_RESEARCH
 Research
 
+PC_RESEARCH_DESC
```
Comment (geoffthemedio) [inline default/stringtables/en.txt:8053]: same re: "It"

File: default/stringtables/en.txt [hunk 22]
```diff
@@ -7994,6 +8058,9 @@ Trade
 PC_PRODUCTION_LOCATION
 Production
 
+PC_PRODUCTION_LOCATION_DESC
+Production parts integrate with a ship to form a [[OBJ_PROD_CENTER]], allowing the creation of at least some types of production projects. The ship must be [[metertype METER_SUPPLY]] connected to a source of [[metertype METER_INDUSTRY]].
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: lose "at least" and "types of"

File: default/stringtables/en.txt (hunk 17 above)
Comment (dbenage-cx) [reply default/stringtables/en.txt]: I've reformatted this article some, along with the other requests.

**Comment by dbenage-cx** [comment]
@geoffthemedio TY for thorough review

File: default/stringtables/en.txt [hunk 23]
```diff
@@ -7917,6 +7920,11 @@ invalid ship part class
 PC_DIRECT_WEAPON
 Direct Weapon
 
+PC_DIRECT_WEAPON_DESC
+'''Direct-fire weapons acquire targets without third-party assistance (a friendly ship or planet).
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: I don't know what this means

File: default/stringtables/en.txt [hunk 24]
```diff
@@ -7917,6 +7920,11 @@ invalid ship part class
...
+
+Most weapons in this category are affected by [[encyclopedia PC_SHIELD]], and some are capable of firing multiple shots per combat round.'''
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: are any not affected by shields?

File: default/stringtables/en.txt [hunk 25]
```diff
@@ -7935,57 +7943,116 @@ Launch Bay
...
-Each launch bay allows a number of fighters to launch each combat round.
-'''
+Launched fighters join combat the round after being launched, returning once combat ends for that turn.'''
 
 # Ship part classes
 PC_SHIELD
 Shields
 
+PC_SHIELD_DESC
+'''Shield part's reduce the [[encyclopedia DAMAGE_TITLE]] sustained from each weapon hit by the shield parts strength.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: parts not part's

File: default/stringtables/en.txt [hunk 26]
```diff
@@ -7935,57 +7943,116 @@ Launch Bay
...
+General purpose and specialized parts that do not fall into any specific category.
+
 # Ship part classes
 PC_BOMBARD
 Bombardment
 
+PC_BOMBARD_DESC
+'''Bombardment parts allow a ship to bombard an enemy planet, typically reducing the population on the planet by the capacity value of any applicable bombardment parts on board.
+
+The majority of bombardment parts only affect one particular [[encyclopedia METABOLISM_TITLE]].'''
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: remove particular

File: default/stringtables/en.txt [hunk 27]
```diff
@@ -7935,57 +7943,116 @@ Launch Bay
...
+
 # Ship part classes
 PC_INDUSTRY
 Industry
 
+PC_INDUSTRY_DESC
+Industry parts increase the [[metertype METER_INDUSTRY]] generated from a ship. The ship must be [[metertype METER_SUPPLY]] connected to a [[OBJ_PROD_CENTER]] (such as a colony) for the output to be utilized. The ship may also require [[metertype METER_POPULATION]] to begin increasing the output.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: still don't know what is meant about needing population.

File: default/stringtables/en.txt (hunk 23 above)
Comment (dbenage-cx) [reply default/stringtables/en.txt]: I assumed `Direct Weapon` referred to the opposite of Indirect-fire, so this was an attempt at a definition without mentioning Line-of-sight.

How about `A Direct Weapon discharges energy or a projectile directly towards a target.` ?

Comment (geoffthemedio) [reply default/stringtables/en.txt]: There are no indirect fire weapons, so you don't need to make a distinction about it in the description. Perhaps "Direct-fire weapons attack and can damage targets each combat round (unlike fighters, which must first be launched before they can attack)."

Comment (dbenage-cx) [reply default/stringtables/en.txt]: Ok, thank you.

File: default/stringtables/en.txt (hunk 24 above)
Comment (dbenage-cx) [reply default/stringtables/en.txt]: Dropped the portion mentioning shields.

File: default/stringtables/en.txt (hunk 27 above)
Comment (dbenage-cx) [reply default/stringtables/en.txt]: Dropped that sentence

File: default/stringtables/en.txt [hunk 28]
```diff
@@ -7935,57 +7943,116 @@ Launch Bay
...
 Colonization
 
+PC_COLONY_DESC
+'''Colony parts integrate with a ship to provide the necessary equipment to establish ownership on a new planet.
+
+These parts may also increase the [[metertype METER_POPULATION]] on a ship.
+When colonizing a planet with a populated colony ship, a colony for the [[encyclopedia ENC_SPECIES]] is established, starting with the same population that was aboard the ship.
+
+A ship with no population will still have enough personnel to establish an [[encyclopedia OUTPOSTS_TITLE]].
+Outposts may construct a colony for any empire species within supply range, provided the source meets other criteria.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: meaning of "empire species" is unclear

Comment (dbenage-cx) [reply default/stringtables/en.txt]: `any empire species` -> `any species in the empire`

Comment (geoffthemedio) [reply default/stringtables/en.txt]: say that then... although it's perhaps misleading since it's not true for species that can't colonize

Comment (dbenage-cx) [reply default/stringtables/en.txt]: Meant that I have changed to such.  This should also link to a `Colony Building` category/article in #1347 , supplementing or replacing `provided the source meets other criteria`.

**Comment by geoffthemedio** [comment]
could probably still be improved, but OK for now

**Pull request merged**